about.html

```diff
@@ -0,0 +1,148 @@
+
+            <div class="hello-write">
+                <p>
+                    The global summit brings together community of experts,
+                    accademics and activists who coprise the creative commons
+                    affiliate network in a different country every two years,
+                    this year we hope to expand our invitation list incluiding
+                    organizations and individuals.
+                </p>
+            </div>
+
+            <div class="hello-contact-div">
+                <div class="hello-contact">
+                    <p class="hello-p">
+                        Please contact us per email for any further questions
+                        about CC global summit 2015!
+                    </p>
+                    <p>chidoziek2@gmail.com</p>
+                </div>
+            </div>
+
+            <div class="creative-commons">
+                <h1>
+                    Creative Commons<br />
+                    Global Summit 2015 Logo
+                </h1>
+            </div>
+
+            <div class="global-line"></div>
+
+            <div class="logo-global">
+                <p>
+                    The logo of CC Global summit 2015 was decided through the
+                    logo competition from 8, june to 7, july
+                </p>
+            </div>
+            <img src="images/about_logo 1.svg" alt="" class="about-logo" />
+
+            <div class="see-summit">
+                <h1>See past Global Summits</h1>
+                <div class="global-line"></div>
+                <p>
+                    Take a look at the past two CC Global summits which took
+                    place in Buenos Aires and in Warsaw
+                </p>
+
+                <div class="see-div">
+                    <div class="first-global">
+                        <h1>2011</h1>
+                        <p>CC Global Summit 2011 in Warsaw</p>
+                    </div>
+                    <div class="first-global">
+                        <h1>2013</h1>
+                        <p>CC Global Summit 2011 in Bueno</p>
+                    </div>
+                </div>
+            </div>
+
+            <div class="first-footer">
+                <h2>Partner</h2>
+                <div class="global-line"></div>
+                <div class="footer-logos">
+                    <img src="images/mozilla 1.svg" alt="" />
+                    <img src="images/googlekorea 1.svg" alt="" />
+                    <img src="images/naver 1.svg" alt="" />
+                    <img src="images/daumkakaologo 1.svg" alt="" />
+                    <img src="images/airbnb 1.svg" alt="" />
+                </div>
+            </div>
+
+            <div class="second-footer">
+                <img src="images/cck-logo1 1.svg" alt="" />
+                <div class="creative-common">
+                    <p>2015 Creative commons Korea Some Rights Reserved</p>
+                </div>
+            </div>
```

Comment:
Great design @Developer-Kingz 🤝

- [ ] As stated in the project requirements, kindly ensure your design matches the design given. Kindly fix the footer of your about section.
- Your design
![Your footer](https://github.com/Developer-Kingz/Microverse-capstone/assets/63917725/c190102a-507f-46d0-862c-96434ecb7e61)
- Given design
![Given Footer](https://github.com/Developer-Kingz/Microverse-capstone/assets/63917725/6e7e712c-247c-4e9d-91ea-c502b4c483ea)